CHANGELOG.md

```diff
@@ -154,6 +154,9 @@ All notable changes to this project will be documented in this file.
 [`explicit_counter_loop`]: https://github.com/Manishearth/rust-clippy/wiki#explicit_counter_loop
 [`explicit_iter_loop`]: https://github.com/Manishearth/rust-clippy/wiki#explicit_iter_loop
 [`extend_from_slice`]: https://github.com/Manishearth/rust-clippy/wiki#extend_from_slice
+[`filter_flat_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_flat_map
+[`filter_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_map
+[`filter_map_flat_map`]: https://github.com/Manishearth/rust-clippy/wiki#filter_map_flat_map
```

Comment:
That's what I think too, but we already have quite a few lints like that (`block_in_if_condition_{expr,stmt}`, `option_map_unwrap_or{,else}`, `{result,option}_unwrap_used`, etc.). Maybe we should merge some of them.